contracts/voting-verifier/src/msg.rs

```diff
@@ -25,6 +26,12 @@ pub enum ExecuteMsg {
     #[permission(Any)]
     VerifyMessages(Vec<Message>),
```

Comment:
is this method used at all for the stacks integration? if not, it should be removed

Reply:
Same as ConstructProof, only ITS Hub calls needs payload, all other calls from Stacks -> other chain go through VerifyMessages.